source/StepBro.Core/Execution/ScriptUtils.cs

```diff
@@ -255,7 +256,7 @@ public static string Find(this ILineReader reader, [Implicit] ICallContext conte
         }
 
         [Public]
-        public static string Await(this ILineReader reader, [Implicit] ICallContext context, string text, TimeSpan timeout, bool skipCurrent = true, bool removeFound = false)
+        public static string Await(this ILineReader reader, [Implicit] ICallContext context, string text, TimeSpan timeout, bool skipCurrent = true, bool removeFound = false, TimeSpan ts = default(TimeSpan))
```

Comment:
I'm adding something similar to this in my pull request (https://github.com/JanJorgensen/StepBro/pull/102/), and as it is not used here, maybe we can wait with adding it until my pull request so we do not get merge conflicts :)